cirq/protocols/json_test_data/_QubitAsQid.json

```diff
@@ -1,8 +1,5 @@
 {
-  "cirq_type": "_QubitAsQid",
-  "qubit": {
-    "cirq_type": "NamedQubit",
-    "name": "a"
-  },
+  "cirq_type": "NamedQid",
+  "name": "a",
   "dimension": 5
-}
+}
```

Comment:
Instead of changing _QubitAsQid to NamedQid, change the test to another type of qubit that doesn't have a qudit variant.  Use a custom qubit type defined just for the test if needed.